08-Image-coords/Image_coordinates_tutorial.ipynb

```diff
@@ -0,0 +1,257 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Handling image coordinates\n",
+    "\n",
+    "The `astropy.wcs` package implements the FITS World Coordinate System (WCS) standard, and some commonly-used distortion conventions used in imaging data.\n",
+    "\n",
+    "This tutorial will show how to extract from FITS files, and how to use it to transform coordinates."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import numpy as np\n",
+    "%matplotlib inline\n",
+    "from matplotlib import pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astropy.utils.data import get_pkg_data_filename\n",
+    "from astropy.io import fits\n",
+    "from astropy import wcs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating a WCS object from the header of a FITS file"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Open a FITS image with `astropy.fits` and examine the contents."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_file = fits.open('data/w5.fits')\n",
+    "\n",
+    "image_file.info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To create a WCS object pass the header with the WCS keywords to astropy.wcs.WCS. In this case it is the primary header. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w = wcs.WCS(image_file[0].header)\n",
+    "w"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For this case, `fits.getheader` may be used to save a step."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w = wcs.WCS(fits.getheader('data/w5.fits'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Check the physical type of each axis"
```

Comment:
I think this is nice to show how to check, but it would be good to explain more (at least if you go over this during the workshop) why one would need to check this and what the "physical type" means.

Reply:
Good point, I expanded the description.